dnainator-javafx/src/main/java/nl/tudelft/dnainator/javafx/controllers/WindowController.java

```diff
@@ -27,32 +32,40 @@
 	private WelcomeController welcomeController;
 	private StrainView strainView;
 	private PhylogeneticView phyloView;
+	private StrainControl strainControl;
 
 	@SuppressWarnings("unused") @FXML
 	private void initialize() {
 		ColorServer colorServer = new ColorServer();
 		fileOpenController.dbPathProperty().set(welcomeController.getListedPaths());
 		fileOpenController.graphProperty().addListener((obj, oldV, newV) -> {
-			strainView = new StrainView(colorServer, newV);
-			constructView();
+			Pane strainView = createStrainView(colorServer, newV);
+			constructView(strainView, phyloView);
 		});
 		fileOpenController.treeProperty().addListener((obj, oldV, newV) -> {
 			phyloView = new PhylogeneticView(colorServer);
 			phyloView.rootProperty().set(newV);
 		});
 		welcomeController.dbProperty().addListener((obj, oldV, newV) -> {
-			strainView = new StrainView(colorServer, newV);
+			Pane strainView = createStrainView(colorServer, newV);
 			phyloView = new PhylogeneticView(colorServer);
-			constructView();
+			constructView(strainView, phyloView);
 		});
 	}
 	
-	private void constructView() {
+	private void constructView(Pane strainView, Pane phyloView) {
 		SplitPane splitPane = new SplitPane(strainView, phyloView);
 		splitPane.setOrientation(Orientation.VERTICAL);
 		root.setCenter(splitPane);
 	}
 
+	private Pane createStrainView(ColorServer colorServer, Graph graph) {
```

Comment:
You don't necessarily need to do this. The minimap I'm working on ATM gets created and added inside `StrainView` itself, which still is just a regular `Pane`. I'm thinking that you can also create `StrainControl` directly from `StrainView` itself. You can take a look at [the minimap](https://github.com/Unia/dnainator/blob/minimap/dnainator-javafx/src/main/java/nl/tudelft/dnainator/javafx/views/StrainView.java#L27) to see how it's done.